src/pages/using-spark/guides/writing-meaningful-web-content-with-semantic-html.mdx

```diff
@@ -0,0 +1,250 @@
+  - `<strong>Your payment is due</strong>` on September 22, 2021.
+- Remember to bring your **driver’s license** to your closing appointment.
+  - Remember to bring your `<strong>driver’s license</strong>` to your closing appointment.
+
+#### The Strong vs The Bold Element vs Styling Text
+
+- Use the **strong element** `<strong></strong>` for important content, and to communicate urgency.
+- Use the <b>bold element</b> `<b></b>` to bring attention to text without indicating that it's more important.
+  - Use for:
+    - Drawing the reader's attention to the elements contents, which aren't of special importance
+    - Keywords in a summary and for product names in a review
+- Use **CSS** (Cascade Styling Sheet) to style or decorate text, like the <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight"> font-weight style property.</a>
+
+<span id="the-emphasis-element" class="docs-b-Link--anchor"></span>
+<SprkHeading
+  element="h3"
+  additionalClasses="sprk-u-mbm sprk-u-Measure"
+  variant="displayThree"
+>
+  The Emphasis Element
+</SprkHeading>
+
+Use the **emphasis** element to give "stressed emphasis" to the enclosed text inside the `<em></em>` tags, in the same way verbal stress works.
+
+- By default, web browsers render the contents inside the emphasis tags as <i>italic type.</i>
+
+- A screen reader will pronounce the words inside the **emphasis** element `<em></em>` tags with an emphasis, using verbal stress.
+
+> As a UX Writer, is important to know when to apply **stress emphasis** to text, because it can **change the meaning** of the sentence.
+
+#### Usage
+
+Use for:
+
+- Words that have a stressed emphasis compared to surrounding text.
+  - For example: Our family <em>loves</em> spending summers on the beach.
+- Changing the meaning of a sentence.
+  - For example: We found the perfect house, let's put an offer <em>
+    now
+      </em>!
+- Referencing a specific instance of microcopy within the web page that users can take action on.
+  - For example: <em>
+    Find Brokers Near Me
+      </em> to connect with a broker in your community.
+
+<span id="examples" class="docs-b-Link--anchor"></span>
+<SprkHeading
+  element="h5"
+  additionalClasses="sprk-u-mbm sprk-u-Measure"
+  variant="displayFive"
+>
+  Examples
+</SprkHeading>
+
+Here are some examples using the em element to give stressed emphasis to some of text.
+
+- Sort by <em>Submitted</em> to see the loans you’ve already completed.
+  - Sort by `<em>Submitted</em>` to see the loans you’ve already completed.
+- Select <em>I don’t have a preference </em> from the dropdown if you change your mind.
+  - Select `<em>I don’t have a preference </em>` from the dropdown if you change your mind.
+
+#### The Emphasis vs Italic or Idiomatic Text Element
+
+- Use the **emphasis element** `<em></em>` to indicate stress emphasis.
+- Use the **italic or idiomatic text element** `<i></i>` for text that is set off from the normal prose, like an alternate tone or mood.
```

Comment:
Idiomatic is a 50cent word that seems to add confusion. I would just skip to Italic.

Reply:
Also, same convo above about strong vs bold vs css weight